### PR TITLE
[Feature Request] Pagination should update URL with page number

### PR DESCRIPTION
**Pull request by martkop12**

### CODE REVIEW

File: src/components/RecipeList.vue [hunk 1]
```diff
@@ -75,6 +75,8 @@ export default {
       return this.items.length;
     },
     paginatedItems() {
+      let url = ( new URL(window.location.href).searchParams.get("page"));
+      if(url != null ? this.pageNumber=url-1 : this.pageNumber=0);
```
Comment (alfg) [inline src/components/RecipeList.vue]: Can you get the query param through `$route`?
```
this.$route.query.page
```

Comment (martkop12) [reply src/components/RecipeList.vue]: Sure, I will have a look when i get home on my laptop :)

Comment (martkop12) [reply src/components/RecipeList.vue]: Used  
> this.$route.query.page 
 to get query parameter

**Comment by martkop12** [comment]
Recheck

**Comment by martkop12** [comment]
recheck

**Comment by martkop12** [comment]
@alfg Could you explain to me what am i doing wrong? Dont understand why the first commit was okay, but not the others

**Comment by alfg** [comment]
It looks like it's one of the component tests failing for `tests/unit/components/RecipeList.spec.js`:
https://travis-ci.org/alfg/opendrinks/jobs/601361568

Don't think it's related to your change.

Can you try syncing up your fork and see if that clears it?

**Comment by martkop12** [comment]
Ok, what should i do with that ? @alfg

**Comment by alfg** [comment]
@martkop12 I'll take care of it! Thanks for the pagination update!

**Pull request merged**